scalableminds/webknossos
# Add LOD mesh support for frontend

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- test local -> add the mesh file with the new format and old format into the l4_sample datasets' segmentation folder `l4_sample/segmentation/meshes/...`. The meshfiles are available in the following slack thread: https://scm.slack.com/archives/C5AKLAV0B/p1674140733622169?thread_ts=1674138004.502719&cid=C5AKLAV0B
- View the l4_sample dataset
- load a mesh with the new meshfile format (precomputed)
- load a mesh with the old meshfile format (precomputed)
- try reloading the meshes
- hide the meshes
- download the meshes
- zoom in and out. all meshes should be always visible.
- delete the meshes
- and more mesh interactions that you can think of :)

### Issues:
- fixes [#6671](https://github.com/scalableminds/webknossos/issues/6671)
- fixes #6907

------
(Please delete unneeded items, merge only when none are left open)
